main.go

```diff
@@ -743,6 +750,13 @@ func clientBackendDialer(tlsConfigSource certloader.TLSConfigSource, network, ad
 		if err != nil {
 			return nil, err
 		}
+		if *clientConnectCertPath != "" {
+			clientCert, err := tls.LoadX509KeyPair(*clientConnectCertPath, *clientConnectKeyPath)
```

Comment:
This means we'd only be able to support PEM files but not other formats like in other flags. Would be good to use the functions for loading certs here analogously to the way it's done for client certificates so we get reloading, etc.